src/app/api/routing.py

```diff
@@ -4,12 +4,14 @@
 from fastapi import HTTPException, Path
 
 from app.api import crud, security
-from app.api.schemas import UserAuth, UserCreation
+from app.api.schemas import UserAuth, UserCreation, UserRead
+from app.api.schemas import AccessCreation, AccessRead, AccessAuth
+from app.api.schemas import DeviceAuth, DeviceCreation, DeviceOut
```

Comment:
I'd only suggest to switch to:
```
from app.api.schemas import (UserAuth, ...)
```

Reply:
With a line for every entity, I find it a bit clearer what we import for each. 
If you really want to wrap everything into one import, I will change it

Reply:
I agree that having a line for each entity is good. But keeping that, we can simply do :
```
from app.api.schema import (UserAuth, UserCreation, UserRead,
                            AccessCreation, AccessRead, AccessAuth,
                            DeviceAuth, DeviceCreation, DeviceOut)
```
to avoid multiple imports :)

Reply:
Ok I updated it